apps/sample-app/src/components/PeopleAvatars.tsx

```diff
@@ -67,7 +61,6 @@ export const PeopleAvatarList: React.FC<AvatarProps> = (props: AvatarProps) => {
     }
   }
   const AvatarExample: React.FunctionComponent = () => {
-    const capabilities = getSupportedCapabilities();
```

Comment:
I believe this was the only place `getSupportedCapabilities` was called from. Can it be removed as part of this change?